src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -99,7 +98,7 @@ public bool HasPdb(string module, out bool embedded)
         public bool EmbeddedPortablePdbHasLocalSource(string module, out string firstNotFoundDocument)
         {
             firstNotFoundDocument = "";
-            using (FileStream moduleStream = File.OpenRead(module))
+            using (FileStream moduleStream = _fileSystem.OpenRead(module) as FileStream)
```

Comment:
we can keep `Stream` remove `as FileStream`, we don't expect null from this api and other api below accept `Stream`

Reply:
Ok done